Assets/Scripts/StaticCanvasManager.cs

```diff
@@ -0,0 +1,38 @@
+using UnityEngine;
+using System.Collections.Generic;
+using System.Linq;
+
+/// <summary>
+/// Manages star content
+/// 
+/// </summary>
```

Comment:
This XML comment is not descriptive of what this class does. Why does it work?

If why it works is already clear from the code, then the XML is redundant (unless we're publishing our API as an engine, which I don't think we're doing). Otherwise, the XML should explain what's going on.